openstates/importers/events.py

```diff
@@ -71,9 +74,24 @@ def get_location(self, location_data: _JsonDict) -> EventLocation:
         )
         return obj
 
+    def get_chamber_name_from_event_name(self, name: str) -> typing.Union[str, None]:
+        if isinstance(name, tuple):
+            possible_chamber_name = name[0].split()[0].lower()
```

Comment:
if `name` can be a `tuple`, I think the typing on the function signature should change (currently `name: str`)

Reply:
Ah, true. Thanks!